web-app/src/invertHex.js

```diff
@@ -0,0 +1,3 @@
+export const invertHex = hex => {
+    return '#' + (Number(`0x1${hex.replace('#','')}`) ^ 0xFFFFFF).toString(16).substr(1).toUpperCase()
+  }
```

Comment:
Why do we need this? (Just curious)

Reply:
Should probably be changed to calculate contrast. I added it as a quick way to get a color for the chip value, to be automatically different to chip color (so it is legible).